### PR TITLE
[4.0] Fix remaining input-group-* classes

### PR DESCRIPTION
**Pull request by C-Lodder**
### Summary of Changes

Fix the remaining `input-group-*` classes that were changed in BS4-beta.3

### CODE REVIEW

File: components/com_finder/tmpl/search/default_form.php [hunk 1]
```diff
@@ -34,7 +34,7 @@
 			</label>
 			<div class="input-group">
 				<input type="text" name="q" class="js-finder-search-query form-control" value="<?php echo $this->escape($this->query->input); ?>">
-				<span class="input-group-btn">
+				<span class="input-group-append">
 				<?php if ($this->escape($this->query->input) != '' || $this->params->get('allow_empty_query')) : ?>
 					<button name="Search" type="submit" class="btn btn-primary">
                         <span class="fa fa-search icon-white"></span>
```
Comment (Quy) [inline components/com_finder/tmpl/search/default_form.php]: Add ` aria-hidden="true"`

File: administrator/components/com_joomlaupdate/tmpl/update/finaliseconfirm.php [hunk 2]
```diff
@@ -26,47 +26,50 @@
...
-				<div class="input-prepend input-append">
-					<span class="input-group-addon">
-						<span class="icon-user hasTooltip" title="<?php echo JText::_('JGLOBAL_USERNAME'); ?>" aria-hidden="true"></span>
-						<label for="mod-login-username" class="sr-only">
-							<?php echo JText::_('JGLOBAL_USERNAME'); ?>
-						</label>
-					</span>
+				<div class="input-group">
 					<input name="username" tabindex="1" id="mod-login-username" type="text" class="form-control" placeholder="<?php echo JText::_('JGLOBAL_USERNAME'); ?>" size="15" autofocus="true">
+					<span class="input-group-append">
```
Comment (wilsonge) [inline administrator/components/com_joomlaupdate/tmpl/update/finaliseconfirm.php:35]: Why are we now appending rather than prepending?

Comment (C-Lodder) [reply administrator/components/com_joomlaupdate/tmpl/update/finaliseconfirm.php:35]: To keep it consistent with the other input fields for the username

Comment (wilsonge) [reply administrator/components/com_joomlaupdate/tmpl/update/finaliseconfirm.php:35]: Makes sense :)

**Pull request merged**